internal/inputs/commands.go

```diff
@@ -105,8 +106,30 @@ func commandRun(dataStore *[]interface{}, yml *load.Config, command load.Command
 
 	// Create the command with our context
 	cmd := buildCommand(ctx, api, command)
+
+	// https://golang.org/pkg/os/exec/#Cmd.StdinPipe
+	if load.Args.StdinPipe {
+		_, err := cmd.StdinPipe()
+		if err != nil {
+			load.Logrus.WithFields(logrus.Fields{
+				"exec":       command.Run,
+				"err":        err,
+				"suggestion": "StdinPipe failed",
+			}).Debug("command: failed")
+		}
+	}
+
 	output, err := cmd.CombinedOutput()
 
+	// check assertion, if the result is false return immediately
```

Comment:
Rathen than comment _what_ is doing could you comment _why_ is doing what is doing.

Reply:
@Noly have updated both the comments on that line, and put further detail above the function itself.